src/lib/public-api.ts

```diff
@@ -4,7 +4,7 @@ export * from './helpers';
 export * from './models';
 export * from './components/editors/ck-editor/presets';
 export * from './utils';
-export * from './route-data';
+export * from './route/data';
```

Comment:
hier sollte dann alles aus der index.ts im `route` ordner exportiert werden.
```suggestion
export * from './route';
```